src/views/catalog/templatescatalog/components/TemplatesCatalogDrawer/hooks/useDrawerContext.tsx

```diff
@@ -103,7 +107,11 @@ const useDrawer = (template: V1Template) => {
     const templateWithRunning = produce(templateWithGeneratedParams, (draftTemplate) => {
       const draftVM = getTemplateVirtualMachineObject(draftTemplate);
 
-      if (isEmpty(draftVM?.spec?.running)) draftVM.spec.runStrategy = RUNSTRATEGY_ALWAYS;
+      if (
+        isEmpty(draftVM?.spec?.running) &&
+        [RUNSTRATEGY_HALTED, RUNSTRATEGY_MANUAL].includes(draftVM?.spec?.runStrategy)
+      )
```

Comment:
Something is not logical here, it can't be both with empty value and include one of the values

Reply:
we are verifying `spec.running` and `spec.runStrategy`.

Reply:
excuse me I thought I saw something different